driver/normalizer/normalizer.go

```diff
@@ -119,63 +140,78 @@ var Normalizers = []Mapping{
 				Fields{
 					{Name: uast.KeyType, Op: Var("ignTypeRet")},
 					{Name: uast.KeyPos, Op: Var("ignPosRet")},
-					//{Name: "IsTypeName", Op: Var("ignIsTypeName")},
-					{Name: "StorageClass", Op: Var("StorageClass")},
-					{Name: "IsVirtual", Optional: "optIsVirtual", Op: Var("ignIsVirtual")},
+					{Name: "IsComplex", Optional: "optIsComplex", Op: Var("ignIsComplex")},
```

Comment:
If I understood correctly, those fields will always appear now, right? If so, please remove `Optional` everywhere.

Reply:
I'm on it, but some are still really `Optional` so not everywhere.